Examples/checkers.lisp

```diff
@@ -0,0 +1,446 @@
+;;;;  Copyright (c) 2020 Daniel Kochmański
+;;;;
+;;;;    License: BSD-2-Clause.
+
+(defpackage #:clim-demo.checkers
+  (:use #:clim-lisp #:clim)
+  (:export #:run-checkers #:clim-checkers)
+  (:import-from #:alexandria
+                #:if-let #:when-let #:when-let* #:iota))
+(in-package #:clim-demo.checkers)
+
+(defparameter +square-side+ 72)
+
+(defparameter +cols+ 8)
+(defparameter +rows+ 8)
+
+(defparameter +col-labels+
+  (mapcar (lambda (n)
+            (format nil "~a" n))
+          (alexandria:iota +cols+ :start 1)))
+
+(defparameter +row-labels+
+  (mapcar (lambda (c)
+            (format nil "~a" (code-char c)))
+          (alexandria:iota +rows+ :start (char-code #\A))))
```

Comment:
Character codes are not guaranteed to work this way. Would `(coerce (subseq "ABCDEF…" 0 +rows+) 'list)` work?

Reply:
if we had a fixed number of labels it would, however the number of rows is configurable (i.e we may have A-Z). I've pushed a function, which does

```
(defparameter *row-labels*
  (map 'list #'princ-to-string
       (if (> *rows* 26)
           (alexandria:iota *rows* :start 1)
           (subseq "ABCDEFGHIJKLMNOPQRSTUVWXYZ" 0 *rows*))))
```

that is, if there is more rows than english alphabet letters, we use numbers.